src/web/impl/AdminVerificationStrategy.cpp

```diff
@@ -40,8 +47,9 @@ PasswordAdminVerificationStrategy::isAdmin(RequestType const& request, std::stri
         // No Authorization header
         return false;
     }
-
-    return it->value() == password_;
+    std::string userAuth(it->value());
+    std::transform(userAuth.begin(), userAuth.end(), userAuth.begin(), ::toupper);
```

Comment:
Should we also have the same transform when we generate `passwordSha256_` just so we don't rely on the hasher to always produce upper-case strings?